bitnami/grafana-operator/values.yaml

```
@@ -545,6 +545,23 @@ grafana:
     timeoutSeconds: 5
     failureThreshold: 6
     successThreshold: 1
+  ## @param extraVolumes Optionally specify extra list of additional volumes for the grafana pod(s)
+  ##
+  extraVolumes: []
+  ## @param extraVolumeMounts Optionally specify extra list of additional volumeMounts for the grafana container(s)
+  ##
+  extraVolumeMounts: []
+  ## @param sidecars Add additional sidecar containers to the grafana pod(s)
```

Could you please add this small change?
```suggestion
  ## @param grafana.extraVolumes Optionally specify extra list of additional volumes for the grafana pod(s)
  ##
  extraVolumes: []
  ## @param grafana.extraVolumeMounts Optionally specify extra list of additional volumeMounts for the grafana container(s)
  ##
  extraVolumeMounts: []
  ## @param grafana.sidecars Add additional sidecar containers to the grafana pod(s)
```